packages/react/stories/Notification.js

```diff
@@ -0,0 +1,81 @@
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { action } from "@storybook/addon-actions";
+import { text, boolean } from "@storybook/addon-knobs/react";
+import { NotificationV1, TextLink } from "../src/hig-react";
```

Comment:
Please use `import { NotificationV1 as Notification, TextLink } from "../src/hig-react";` to minimize the effort needed to switch to a different version.